draft-wang-ppm-differential-privacy.md

```diff
@@ -239,11 +254,128 @@ The following considerations are out-of-scope for this document:
 
 This document uses the same conventions for error handling as {{!DAP}}.
 
-> TODO: add more
+Let `exp(EPSILON)` denote raising the numeric constant `e` to the power of
+`EPSILON`.
+
+# Security Goals and Trust Model {#overview}
+
+## Differential Privacy
+
+DP formalizes a property of any randomized algorithm that takes in a sequence
+of measurements, aggregates them, and outputs an aggregate result. Intuitively,
+this property guarantees that no single measurement signficantly impacts the
+value of the aggregate result. This intuition is formalized by {{DMNS06}} as
+follows.
+
+> CP: The following is might be too jargony for an informative RFC, but for now
+> I think we're all just trying to agree on the definition. Once we have
+> consensus among ourselves, we can punt this to the appendix and leave a less
+> formal description here.
+
+[CP: This is substitution+pure-DP: Is {{DMNS06}} the right reference?] We say
+that two sequences of measurements `D1` and `D2` are "neighboring" if they are
+the same length and contain all the same measurements except one (i.e., the
+difference between their multisets contains one element).
+
+Let `p(S, D, R)` denote the probability that randomized algorithm `S`, on input
+of measurements `D`, outputs aggregate result `R`.
 
-# Security Goals and Threat Model {#overview}
+A randomized algorithm `R` is called "`EPSILON`-DP" if for all neighboring `D1`
```

Comment:
R and S are mixed up

Reply:
Good catch. Previously I was using `R` for the algorithm and missed this one.